src/query/network/network.hooks.ts

```diff
@@ -0,0 +1,8 @@
+import { useGetNetworkStatus } from './network.query';
+
+export function useNetworkStatus(url: string): boolean {
+  let isOnline: boolean = false;
+  const result = useGetNetworkStatus(url);
+  if (result.isSuccess) return (isOnline = true);
+  return isOnline;
+}
```

Comment:
isn't this the same as: 

```suggestion
export function useNetworkStatus(url: string): boolean {
  const result = useGetNetworkStatus(url);
  return result.isSuccess;
}
```